public/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset='utf-8'>
+    <title>Reviews</title>
+    <link href='main.css' rel='stylesheet'>
+  </head>
+  <body>
+    <div id='app'></div>
+    <script src='./app.js'></script>
+  </body>
+</html>
```

Comment:
@MarissaShieh Please add a newline break at the end of the file to meet style guide.